CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 * ![Bugfix][badge-bugfix] `@repl` blocks now work correctly together with quoted
   expressions. ([#923][github-923], [#926][github-926])
 
+* ![Bugfix][badge-bugfix] `@example`, `@repl` and `@eval` blocks now handle keywords,
```

Comment:
Maybe stupid, but how are `try` / `catch` keywords?

Reply:
They are called `reserved words` in the parser.